recipes/homer/meta.yaml

```diff
@@ -5,21 +5,33 @@ about:
 
 package:
   name: homer
-  version: '4.8'
+  version: '4.8.3'
 source:
   fn: configureHomer.pl
   md5: f0f52cfe22bfe341532a11d947aac803
   url: http://homer.salk.edu/homer/configureHomer.pl
 build:
-  number: 1
+  number: 3
```

Comment:
It's already merged so leave it like that, but next time if you increase the version number the build number needs to start with 0 :)